src/mon/OSDMonitor.cc

```diff
@@ -4384,6 +4384,22 @@ void OSDMonitor::get_pools_health(
 	detail->push_back(make_pair(HEALTH_WARN, ss.str()));
     }
 
+    if (1 == pool.size) {
+      stringstream ss;
+      ss << "pool '" << pool_name << "' size is 1 which not safe enough ";
+      summary.push_back(make_pair(HEALTH_WARN, ss.str()));
+      if (detail)
+        detail->push_back(make_pair(HEALTH_WARN, ss.str()));
+    }
+
+    if (1 == pool.min_size) {
```

Comment:
sorry, it's not a repetition; it's checking a different variable. It does seem though that you could use the same block for both though.